CoffeeChat/Controller/SchedulingPlacesViewController.swift

```diff
@@ -92,12 +114,29 @@ class SchedulingPlacesViewController: UIViewController {
         titleLabel.textColor = .textBlack
         view.addSubview(titleLabel)
 
-        infoLabel.font = ._16CircularStdBook
-        infoLabel.text = "Pick three"
-        infoLabel.textColor = .textLightGray
+        var scheduledTime = ""
+        if let confirmedTime = confirmedTime.first {
+            let amTimes = ["9:00", "9:30", "10:00", "10:30", "11:00", "11:30"]
```

Comment:
I feel like we can define this elsewhere outside of this function? Cause we might want to do things with like knowing if a time is AM or PM in the future that isn't for this particular screen or case.

Reply:
Like the amTimes and pmTimes

Reply:
+1, maybe make a `Time+Helper.swift` or some utils file similar to that with `getAMTimes()/getPMTimes()`